.tekton/operator-bundle-pipeline.yaml

```diff
@@ -51,6 +51,9 @@ spec:
   - description: Output Image Repository
     name: output-image-repo
     type: string
+  - name: operator-image-repo
+    description: Image repository for the StackRox Operator.
+    type: string
```

Comment:
[minor] Anticipating that we'll have to define a parameter for each operand image, could you please move this parameter to the end of the list so that we can append others closer to it?

Reply:
done